app/views/software_records/edit_road_map.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="container animated fadeIn" style="opacity: 0.8;">
+  <div class="card-detail bg-dark text-white p-4">
+    <hr class="bg-white">
+    
+    <h1 class="text-center" style="color: white;">Edit Road Map</h1>
+
+    <%= form_with(model: @software_record, url: update_road_map_software_record_path(@software_record), local: true, class: 'mt-4') do |form| %>
+      <% if @software_record.errors.any? %>
+        <div id="error_explanation" class="alert alert-danger">
+          <h2><%= pluralize(@software_record.errors.count, "error") %> prohibited this road map from being saved:</h2>
+          <ul>
+            <% @software_record.errors.full_messages.each do |message| %>
+              <li><%= message %></li>
+            <% end %>
+          </ul>
+        </div>
+      <% end %>
+
+      <% if !@software_record.title.nil? && !@software_record.title.to_s.empty? %> 
+        <div class="form-group">
+          <%= form.label :title, class: 'form-label' %>
+          <p class="form-control-plaintext text-white"><%= @software_record.title %></p> <!-- Display the title but don't allow editing -->
+        </div>
+      <% end %>
+
+      <div class="form-group">
+        <%= form.label :road_map, class: 'form-label' %>
+        <%= form.text_area :road_map, class: 'form-control' %>
```

Comment:
Set the the road map field on the edit_road_map view to be 10 rows tall so we have a big field to see lots of text when editing.